css/components/_passage.css

```diff
@@ -1,21 +1,115 @@
+.top-nav {
+  top: 0;
+
+  text-align: center;
+
+  background-color: $vintage-background;
+
+  .meta-information {
+    margin-top: 1em;
+
+    font-size: 1.25em;
+
+    color: $vintage-gray;
+  }
+}
+
+.stage-controls {
+  bottom: 0;
+
+  display: flex;
+  box-shadow: 0 0 60px 10px $vintage-background;
+
+  background-color: rgba($vintage-background-rgb, .75);
+}
+
+.top-nav,
+.stage-controls {
+  position: fixed;
+  right: 0;
+  left: 0;
+  z-index: 2;
+
+  width: 100%;
+  max-width: $large-width;
+  padding: $default-padding;
+  margin: 0 auto;
+  box-sizing: border-box;
+
+  button {
+    display: inline-block;
+
+    background: $vintage-eggy;
+		border: 0;
+    border-radius: $corner-radius;
+    color: $vintage-white;
+
+    flex-grow: 1;
+    margin: 0 0.5em;
+    text-align: center;
+
+    user-select: none;
+
+    i.fa {
+      width: 2em;
+    }
+
+    &.active {
+      background: $vintage-rust;
+    }
+
+    &:focus {
+      outline: 0;
+    }
+
+    &:first-child {
+      margin-left: 0;
+    }
+
+    &:last-child {
+      margin-right: 0;
```

Comment:
This is working correctly when the audio player is turned on because that becomes that last element.  May just need to switch the order of the player and button in the AudioPlayer component